reactor-netty-http/src/main/java/reactor/netty/http/server/HttpServerRequest.java

```diff
@@ -143,6 +144,13 @@ default Flux<HttpContent> receiveContent() {
 	 */
 	HttpHeaders requestHeaders();
 
+	/**
+	 * return parsed and decoded query parameter name value pairs
+	 *
+	 * @return query parameters {@link Map&lt;String,List&lt;String&gt;&gt;}
+	 */
```

Comment:
```suggestion
	 * @return query parameters {@link Map}
	 */
```

To avoid some compilation warnings, I think it is safe to just remove the generic types in the javadoc link.